MODULE.bazel

```
@@ -0,0 +1,98 @@
+"""Defines the bzlmod module"""
+
+module(
+    name = "swift_bep_parser",
+    version = "0.0.0",
+    bazel_compatibility = [
+        ">=6.0.0",
+        "<7.0.0",
```

Any thoughts about making this `< 8.0.0`, @luispadron?
```suggestion
        "<8.0.0",
```

---

I wasn't sure if BEP was compatible between 6 & 7 but if it is that makes sense!

---

We're technically using the BEP definition from Bazel 8 (AFAIK) so let's do it.